src/lib/blockchain_snark/blockchain_snark_state.ml

```diff
@@ -311,7 +311,7 @@ let rule ~proof_level ~constraint_constants transaction_snark self :
             (step ~proof_level ~constraint_constants ~logger:(Logger.create ())
                [ x1; x2 ] x )
         in
-        ([ b1; b2 ], ()) )
+        ([ b1; b2 ], (), ()) )
```

Comment:
nit: I know this was inherited from the previous code and also there needs to be limits of how much can be done in a PR, so just noting this for future.

If these are base proofs, variable names like `base_proof_1` and `base_proof_2` would make understanding this code a lot easier.